apps/site/lib/site_web/ambiguous_alert.ex

```diff
@@ -0,0 +1,63 @@
+defprotocol AmbiguousAlert do
```

Comment:
nitpick: given the filesystem location, should this be `SiteWeb.AmbiguousAlert`?